File: app/src/main/java/com/example/user/eventest/MainPresenter.java
package com.example.user.eventest;

import com.example.user.eventest.model.Memo;

/**
 * Created by DR
 * on 26.02.2018.
 */

public interface MainPresenter {
    void fabClick();

    void setSelectedMemoToEdit(Memo item);

    void menuEditClick();

    void showNewMemoOnStart();
}

File: app/src/main/java/com/example/user/eventest/MainPresenter.kt
package com.example.user.eventest

import com.example.user.eventest.model.Attachments
import com.example.user.eventest.model.Memo

/**
 * Created by DR
 * on 26.02.2018.
 */

interface MainPresenter {
    fun fabClick()
    fun setSelectedMemoToEdit(memo: Memo)
    fun menuEditClick()
    fun showNewMemoOnStart()
    fun getAttachments(memoID: Long): Attachments?
}
